tests/plugins/test_ddp_module_wrapper.py

```diff
@@ -0,0 +1,79 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
@ananthsub  Yeah. I've thought about adding test cases directly in these corresponding plugins, but I created a standalone test because I feel like wrapper change is a feature shared by multiple plugins, it's better to let the testing standalone to avoid scattering around. What do you think? I can move the test cases back to individual plugin.

Reply:
I slightly prefer keeping them in the individual plugins because this behavior isn't a general one across plugins. It is specific to this method of data-parallel training which assumes parameters aren't distributed across gpus